cli/src/main/java/com/devonfw/tools/ide/process/ProcessContextImpl.java

```diff
@@ -304,6 +304,7 @@ private String addExecutable(String exec, List<String> args) {
     }
     if ("msi".equalsIgnoreCase(fileExtension)) {
       args.add(0, "/i");
+      args.add(0, "/qb");
```

Comment:
https://docs.revenera.com/installshield25helplib/helplibrary/IHelpCmdLineMSI.htm
> qb creates a basic user interface

might fix the problem but in case we need to run an installer for a global tool like docker why don't we just run that in `BACKGROUND`? I do not know if this `/qb` option fixes this bug but adds another bug to some other tool using an MSI...

Reply:
revert